fiware-region-sanity-tests/tests/fiware_region_object_storage_tests.py

```diff
@@ -31,6 +31,12 @@
 
 class FiwareRegionsObjectStorageTests(fiware_region_base_tests.FiwareRegionsBaseTests):
```

Comment:
Please use ``from fiware_region_base_tests import FiwareRegionsBaseTests``, so that we don't use this long name here.